widgets/S3FileCopySink-batchsink.json

```diff
@@ -0,0 +1,93 @@
+{
+  "metadata": {
+    "spec-version": "1.3"
+  },
+  "configuration-groups": [
+    {
+      "label": "S3 File Copy Sink",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "URI",
+          "name": "filesystemURI"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Base Path",
+          "name": "basePath"
+        },
+        {
+          "widget-type": "select",
+          "label": "Enable Overwrite",
+          "name": "enableOverwrite",
+          "widget-attributes": {
+            "values": [
+              "true",
+              "false"
+            ],
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "select",
+          "label": "Preserve File Owner",
+          "name": "preserveFileOwner",
+          "widget-attributes": {
+            "values": [
+              "true",
+              "false"
+            ],
+            "default": "false"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Buffer Size (MB)",
+          "name": "bufferSize",
+          "default": "1"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Access Key ID",
+          "name": "accessKeyId"
+        },
+        {
+          "widget-type": "password",
+          "label": "Secret Key ID",
+          "name": "secretKeyId"
+        },
+        {
+          "widget-type": "select",
+          "label": "Amazon Region",
```

Comment:
Is the amazon region set likely to grow?
If so, one way to handle that is to allow a textbox for user to enter in the region if it doesn't exist in this dropdown.